app/src/main/res/layout/fragment_patient_chart.xml

```diff
@@ -86,4 +86,11 @@
       android:layout_height="fill_parent"
       android:layout_below="@+id/patient_chart_last_update_section"/>
 
-</RelativeLayout>
+  <ListView
+    android:id="@+id/chart_list"
+    android:layout_width="200dp"
```

Comment:
can you specify this value and the background color in `dimens.xml` and `colors.xml` respectively?